src/formats/kml/geom/KmlPolygon.js

```diff
@@ -206,6 +211,16 @@ define([
         return locations;
     };
 
+    /**
+     * @inheritDoc
+     */
+    KmlPolygon.prototype.isValidAltitudeMode = function (altMode) {
```

Comment:
Lets do this like this:
    KmlPolygon.prototype.isValidAltitudeMode = function (altMode) {
        return WorldWind.CLAMP_TO_GROUND === altMode
            || WorldWind.RELATIVE_TO_GROUND === altMode
            || WorldWind.ABSOLUTE === altMode;

    };